stix_shifter_modules/guardium/stix_translation/query_constructor.py.org

```diff
@@ -0,0 +1,249 @@
+from stix_shifter_utils.stix_translation.src.patterns.pattern_objects import ObservationExpression, ComparisonExpression, \
```

Comment:
Were you planning on keeping or deleting this file? You have two query_constructors.